### DISCUSSION TITLE
如何给全局的mete定义ts类型

### DISCUSSION THREAD

**Discussion by songpeng154** [alovajs/alova#335] [Q&A]
### 请详细描述bug

mete在请求拦截器和响应拦截器中没有代码提示，希望能类似与axios一样，能扩展自定义配置。
![image](https://github.com/alovajs/alova/assets/73383713/915dc473-84a1-466c-bff8-1d15f75e78f7)

### 复现步骤

-

### alova 版本

2.18..0

**Comment by MeetinaXD** [comment]
非bug请在讨论区讨论。

请参考：#333
